common/app/model/dotcomrendering/DotcomRenderingSupportTypes.scala

```diff
@@ -24,18 +23,21 @@ case class Tag(
     title: String,
     twitterHandle: Option[String],
     bylineImageUrl: Option[String],
+    bylineLargeImageUrl: Option[String],
 )
 
 object Tag {
   implicit val writes = Json.writes[Tag]
 
+  // TODO: In second PR we will set bylineImageUrl to t.properties.bylineImageUrl.map(src => ImgSrc(src, Item300)),
```

Comment:
What is the reason for wanting to make this change in a second PR?

Reply:
So we don't break the current implementation and the site works as expectesd. Once the DCR code change is made, we can flip this out. 

It would be 30 minutes of brokenness, but 0 minutes is better.

Reply:
Ah I'm with you now :) Sounds good!